app/services/pdf_parsers/umc_pathology_parser.rb

```diff
@@ -0,0 +1,125 @@
+module PdfParsers
+  class UmcPathologyParser
+    def initialize(pdf_path)
+      @pdf_path = pdf_path
+      @processed_data = {
```

Comment:
This differs a lot from the same [property](https://github.com/Code-the-Dream-School/health-keeper/pull/40/files/474dc64e5aef464e20c369963f9718dee4d8311e#diff-09a18241945656e739cb7fa8113bc2e62c4befc22bf79999e9f4cb4878b43d39R46) of Sterling Accuris parser.